### PR TITLE
Div 3502 resp defend await answer

### PR DESCRIPTION
**Pull request by mounikahmcts**
# Description

When ccd state is AosSubmittedAwaitingAnswer then progress bar should give relevant info and upon continuing should show respondent response in view only mode.

Fixes # DIV-3502

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:

* Hardware:
* O/S and version:
* JDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: test/e2e/journeys/AosSubmittedAwaitingAnswer.test.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      }));
+  });
+  after(() => {
+    request.get.restore();
+  });
+  journey.test([
+    { step: Start },
+    { step: IdamLogin, body: { success: 'yes' } },
+    { step: petitionProgressBar },
+    { step: reviewAosResponse }
```
Comment (dtaylor7) [inline test/e2e/journeys/AosSubmittedAwaitingAnswer.test.js]: would you not want to test through to submission?

Also checkout this change in my PR that verifies data submitted to CCD/COS. Good for making sure all the correct data is collected and sent: https://github.com/hmcts/div-decree-nisi-frontend/pull/118/files#diff-059e3ce21fd547cbb9aabf6f687d2742R69

Comment (mounikahmcts) [reply test/e2e/journeys/AosSubmittedAwaitingAnswer.test.js]: > would you not want to test through to submission?
> 
> Also checkout this change in my PR that verifies data submitted to CCD/COS. Good for making sure all the correct data is collected and sent: https://github.com/hmcts/div-decree-nisi-frontend/pull/118/files#diff-059e3ce21fd547cbb9aabf6f687d2742R69

--> ReviewAosResponse becomes readOnly in this case and will be the last page for the user until the state changes to DNAwaiting. so, e2e for this state "AosSubmittedAwaitingAnswer" is only these 4 steps and he cannot proceed further to submission

**Comment by mounikahmcts** [comment]
Changes are going as part of https://github.com/hmcts/div-decree-nisi-frontend/pull/136

**Comment by mounikahmcts** [comment]
Closing this PR as https://github.com/hmcts/div-decree-nisi-frontend/pull/136 has same changes

**Pull request closed without merging**